src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -1323,9 +1323,10 @@ def parameter_estimation_callback(y):
         p0 = []
         for i,parameter in enumerate(self.parameters):
             p0.append(self.model.get(parameter)/self.options["scaling"][i])
+        p0 = N.array(p0).flatten()
```

Comment:
Doesn't this change affect the line 1331 further down? Since now we give `sciopt.minimize` a flattened array but previously we did not, maybe it doesn't matter?